sfms/config.ini.example

```diff
@@ -1,5 +1,6 @@
 secret=secret
```

Comment:
```suggestion
secret=somesecret
```

Reply:
Recommending this change to make it consistent with the default `SFMS_SECRET` we have in env.example